utils/oath-toolkit/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oath-toolkit
+PKG_VERSION:=2.6.2
+PKG_RELEASE:=1
+PKG_SOURCE:=oath-toolkit-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/oath-toolkit/
+PKG_SOURCE_MD5SUM:=4a05cd4768764843bd5493609a6bdb17
+PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Fam Zheng <fam@euphon.net>
+
+include $(INCLUDE_DIR)/package.mk
+
+CONFIGURE_ARGS += \
+	--disable-xmltest \
+	--disable-pskc
+
+define Package/oath-toolkit
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Oath toolkit
+  URL:=http://www.nongnu.org/oath-toolkit/index.html
+  DEPENDS:=
+endef
+
+define Package/oath-toolkit/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/oathtool/.libs/oathtool $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/liboath/.libs/liboath.so* $(1)/usr/lib/
```

Comment:
While copying binaries from `.libs/` may work, I would recommend setting `PKG_INSTALL:=1` and copy from `PKG_INSTALL_DIR` instead.